automation/automation/expr_types.go

```diff
@@ -29,3 +32,25 @@ func CastToEmailMessage(val interface{}) (out *emailMessage, err error) {
 		return nil, fmt.Errorf("unable to cast type %T to %T", val, out)
 	}
 }
+
+func ReadRequestBody(in interface{}) (s string) {
+	var (
+		b   []byte
+		err error
+	)
+
+	switch val := in.(type) {
+	case *http.Request:
+		b, err = io.ReadAll(val.Body)
+	case io.Reader:
+		b, err = io.ReadAll(val)
```

Comment:
If `val` would be one of the expanded IO interfaces (`io.ReadCloser`, `io.ReadCloseSeeker`, ...) would this case still catch them (I'm not entirely sure)?
Can those even happen? I would personally handle them in a better safe than sorry kind of thing.

Reply:
`io.ReadCloser` is a `io.Reader` and a `io.Closer`, so it would catch that